app/javascript/components/Articles/ArticlesList.tsx

```diff
@@ -1,11 +1,13 @@
 import React, { useState, useEffect } from "react";
 import ArticlesDetails from './ArticlesDetails';
-import { array } from "prop-types";
 
 export interface ArticlesListProps {
+    currentUser:any,
     articles?:any
 }
 
+// console.log(props.articles.map((article, index) => typeof(article.createdAt))
```

Comment:
Commented out console.log should probably be removed before merging.

Reply:
Oops 🙃